sysutils/fzf/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           golang 1.0
 
 go.setup            github.com/junegunn/fzf 0.27.3
```

Comment:
The version number should be `0.28.0` now as it was recently updated in 4cc1dd12f28ca6ba6083ee9f953dfa56a0d02b40.

Reply:
Thanks. Rebased onto 0.28.0 and changed the notes.